tide_core.module

```diff
@@ -152,6 +152,28 @@ function tide_core_form_alter(&$form, FormStateInterface $form_state, $form_id)
       }
     }
   }
+  // Re-order the user_form field.
+  if ($form_id === 'user_form') {
```

Comment:
Hi @MdNadimHossain 
This is the new code for re-order the fields.
Thanks

Reply:
why we are manipulating the weight in the alter hook, why can't we just make the changes in the config file directly using the update hook and then export the settings ?

Reply:
Hi @MdNadimHossain 

Good question, I think so too, but the fields are defined by code as 'base field', without the field setting UI. Please refer to the images below for more detail.
<img width="1718" alt="Screen Shot 2022-11-10 at 1 42 49 pm" src="https://user-images.githubusercontent.com/6699227/200990750-d7647622-624a-4e54-82ab-f9617bb370c7.png">
The fields are grouped into a fieldset $form['account'], so custom fields can't add to the group by UI
'status', 'roles', 'notify', and  'password_policy_status' fields even doesn't show in the form settings
'notify', and  'password_policy_status' was added by contributor modules, they do not exist in all sites so we need code to check if the field is exist
That's why we need a custom code to set field orders correctly.
Thank you.
Henry